Nix conversion

max-privatevoid opened the pull request:
Fixes #1

max-privatevoid committed:
add basic flake structure

max-privatevoid committed:
flake.lock: Add

Flake lock file updates:

• Added input 'flake-parts':
    'github:hercules-ci/flake-parts/7c7a8bce3dffe71203dcd4276504d1cb49dfe05f' (2023-01-26)
• Added input 'flake-parts/nixpkgs-lib':
    follows 'nixpkgs'
• Added input 'nixpkgs':
    'github:NixOS/nixpkgs/9b97ad7b4330aacda9b2343396eb3df8a853b4fc' (2023-01-25)

max-privatevoid committed:
add gitignore

max-privatevoid committed:
basic gui login

max-privatevoid committed:
basic user configuration

max-privatevoid committed:
add hyprland input

Flake lock file updates:

• Added input 'hyprland':
    'github:hyprwm/Hyprland/94b7b6b58475a0e802e390591f95e04c70aa676d' (2023-01-26)
• Added input 'hyprland/hyprland-protocols':
    'github:hyprwm/hyprland-protocols/b8f55e02a328c47ed373133c52483bbfa20a1b75' (2022-12-23)
• Added input 'hyprland/hyprland-protocols/nixpkgs':
    follows 'hyprland/nixpkgs'
• Added input 'hyprland/nixpkgs':
    follows 'nixpkgs'
• Added input 'hyprland/wlroots':
    'gitlab:wlroots/wlroots/5f264a7d6c8af27d41ff440c05262b022c055593' (2023-01-04)
• Added input 'hyprland/xdph':
    'github:hyprwm/xdg-desktop-portal-hyprland/d479c846531fd0e1d2357c9588b8310a2b859ef2' (2023-01-07)
• Added input 'hyprland/xdph/hyprland-protocols':
    follows 'hyprland/hyprland-protocols'
• Added input 'hyprland/xdph/nixpkgs':
    follows 'hyprland/nixpkgs'

max-privatevoid committed:
import hyprland nixos module

max-privatevoid committed:
basic hyprland setup

max-privatevoid committed:
firefox setup

supraBlatt reviewed (commented):
imagine firefox actually working with my dual GPU setup

max-privatevoid committed:
add home-manager input

Flake lock file updates:

• Added input 'home-manager':
    'github:nix-community/home-manager/bb4b25b302dbf0f527f190461b080b5262871756' (2023-01-26)
• Added input 'home-manager/nixpkgs':
    follows 'nixpkgs'
• Added input 'home-manager/utils':
    'github:numtide/flake-utils/5aed5285a952e0b949eb3ba02c12fa4fcfef535f' (2022-11-02)

max-privatevoid committed:
import home-manager nixos module

max-privatevoid committed:
home-manager configuration, configure hyprland

max-privatevoid committed:
install some utilities

max-privatevoid committed:
port hyprland autostart

max-privatevoid committed:
configure kitty

max-privatevoid committed:
install fonts

max-privatevoid committed:
configure nix

max-privatevoid committed:
add nixpkgs to system flake registry
